cli/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "starknet-rs-cli"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["with_mimalloc"]
+with_mimalloc = ["mimalloc"]
+
+[dependencies]
+cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo", rev = "4afacfd574de1121d61dea28f8c73e1516c7e07d"}
+cairo-lang-casm = { git = "https://github.com/starkware-libs/cairo", rev = "4afacfd574de1121d61dea28f8c73e1516c7e07d"}
+cairo-vm = { git = "https://github.com/lambdaclass/cairo-rs", features = ["cairo-1-hints"] }
+getset = "0.1.2"
+lazy_static = "1.4.0"
+num-bigint = { version = "0.4", features = ["serde"] }
+num-integer = "0.1.45"
+num-traits = "0.2.15"
+serde = { version = "1.0.152", features = ["derive"] }
+serde_json = { version = "1.0", features = ["arbitrary_precision"] }
+sha3 = "0.10.1"
+starknet_api = { git = "https://github.com/lambdaclass/starknet-api", rev="b29d881", features = [
```

Comment:
Instead of using the lambdaclass fork of this lib you should use the one from https://github.com/starkware-libs/starknet-api